### ISSUE TITLE
The demo on gh-pages does not work anymore

### ISSUE THREAD

**Issue by artus** [chuckha/ColorFlood#5]
It redirects to a survey spam/survey website. Was your code hijacked?

**Comment by chuckha** [comment]
Nah, my domain expired. I'll remove the link and potentially update to make it work again. But definitely disabling that link for now.
